src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanel.kt

```diff
@@ -344,6 +336,18 @@ class TestCasePanelBuilder(
         })
     }
 
+    private fun updateErrorRelatedUI() {
+        updateBorder(error)
+        if (error.isNullOrBlank()) {
+            errorLabel.isVisible = false
+        } else {
+            errorLabel.isVisible = true
+            errorLabel.toolTipText = error
```

Comment:
I do not know how the render process works for IntelliJ plugins, but if it is reactive, then here exists a moment when the `errorLabel` is visible but yet contains no error message. In this case, it is advisable to swap the assignments to:
```kotlin
errorLabel.toolTipText = error
errorLabel.isVisible = true
```